<launch>
    <node pkg="marker_detector" type="marker_detector" name="marker_detector" output="screen">
        <!--param name= "cam_cal_file" type="string" value="$(find marker_detector)/config/camera.yaml"/-->
	<param name= "cam_cal_file" type="string" value="$(find marker_detector)/config/camera_color.yaml"/>
	<remap from="~image" to="/djiros/image"/>
	<remap from="~camera_pose" to="/vins_estimator/camera_pose"/>
    </node>
</launch>
